$NetBSD: patch-configure,v 1.1.2.2 2017/04/23 10:13:36 bsiegert Exp $

make sure that the --version output of flex is parsed correctly to extract
the version number even if it contains more than one space character
(this is the case on MacOS X El Capitan, for example)

--- configure.orig	2017-04-05 23:38:48.000000000 +0200
+++ configure	2017-04-05 23:39:32.000000000 +0200
@@ -21304,7 +21304,7 @@
   fi
 
         flex_min_version=2.5.31
-  flex_version=`$FLEX_PATH --version | head -n 1 | sed 's/^.* //' | sed 's/[a-zA-Z]*$//' | cut -d' ' -f1`
+  flex_version=`$FLEX_PATH --version | head -n 1 | sed 's/^[^ ]* //' | sed 's/[a-zA-Z]*$//' | cut -d' ' -f1`
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking flex version $flex_version >= $flex_min_version" >&5
 $as_echo_n "checking flex version $flex_version >= $flex_min_version... " >&6; }
   if perl -w <<EOF
